recipes/django-bootstrap5/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "django-bootstrap5" %}
+{% set version = "21.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cb7cbc1582e90718be7ba8e19cdafa47a5bbb102de9c7855b1a1299bae162c1d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.8
```

Comment:
I think you could add `importlib-metadata` as run requirements if you want to support python 3.6 and 3.7.
But not sure not sure why there is a constraint to `<3` in the `setup.py`...

If you are happy with `>=3.8` (3.6 is soon EOL), I'd put the same constraint for the host python requirement.

Reply:
I'm happy with the 3.8 requirements. Updated host requirement as suggested.